Sources/HipMobileUI/ViewModels/Views/ExhibitPreviewViewModel.cs

```diff
@@ -52,7 +52,7 @@ void Accept()
         {
             MessagingCenter.Send<NavigationViewModel, bool>(this, "ReturnValue", true);
             Navigation.ClearModalStack();
-            Navigation.PushAsync(new AppetizerPageViewModel(exhibit));
+            Navigation.PushAsync(new AppetizerViewModel(exhibit));
```

Comment:
Same here, does this work? There should be an `await` here or the interface should be changed s.t. it does not return a `Task` anymore.

Reply:
These changes are from some previous commit. I just changed the name. for fixing ReSharper build error. 
Need to check who made this and why??